css-align-3/default-alignment/place-items-shorthand-004.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<title>CSS Box Alignment: place-items shorthand - invalid values</title>
+<link rel="author" title="Javier Fernandez" href="mailto:jfernandez@igalia.com" />
+<link rel="help" href="http://www.w3.org/TR/css3-align/#place-items-property" />
+<meta name="assert" content="Check that place-items's invalid values are properly detected at parsing time." />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+    #test {
+        align-items: start;
+        justify-items: end;
+    }
+</style>
+<div id="log"></div>
+<div id="test"></div>
+<script>
+    function checkInvalidValues(value) {
+        var div = document.getElementById("test");
+        div.setAttribute("style", "place-items: " + value);
+        var style = getComputedStyle(div);
+        assert_equals(style.getPropertyValue("align-items"),
+            "start", "align-items computed value");
+        assert_equals(style.getPropertyValue("justify-items"),
+            "end", "justify-items computed value");
+    }
+
+    test(function() {
+        checkInvalidValues("center safe")
+        checkInvalidValues("true center")
+    }, "Verify overflow keywords are invalid");
+
+    test(function() {
+        checkInvalidValues("center space-between start")
+    }, "Verify fallback values are invalid");
+
+    test(function() {
+        checkInvalidValues("10px left")
+        checkInvalidValues("right 10%")
+    }, "Verify numeric values are invalid");
+
+    test(function() {
+       checkInvalidValues("auto right")
+       checkInvalidValues("auto auto")
+    }, "Verify 'auto' values are invalid");
```

Comment:
Checking the spec this is right as `auto` is only valid for `justify-items`.
But it'd be nice to add that option in the other test, so we verify that `auto` works for `justify-items`.